src/main/java6/net/finmath/marketdata/model/curves/Curve.java

```diff
@@ -493,4 +486,12 @@ public CurveBuilderInterface getCloneBuilder() throws CloneNotSupportedException
 		CurveBuilder curveBuilder = new CurveBuilder(this);
 		return curveBuilder;
 	}
-}
+	
+	@Override
+	public String toString() {
+		String pointsAsString = "";
+		for (Point point : points)
+			pointsAsString += "\n[" + Schedule.getDateFromDouble(getReferenceDate(),point.time) + ": " + valueFromInterpolationEntity(point.value, point.time) + "]";

```

Comment:
Unit Tests are failing! The unit test curves.CalibrationTest fails here with a NPE! It is not guaranteed that getReferenceDate ist not null!
:-(